docs/en/latest/FAQ.md

```diff
@@ -59,7 +59,7 @@ For the configuration center, configuration storage is only the most basic funct
 4. Change Notification
 5. High Performance
 
-See more [etcd why](https://github.com/etcd-io/etcd/blob/master/Documentation/learning/why.md#comparison-chart).
+See more [etcd why](https://github.com/etcd-io/website/blob/master/content/docs/v3.4/learning/why.md#comparison-chart).
```

Comment:
I think we can use the `next` version doc.

https://github.com/etcd-io/website/blob/master/content/docs/next/learning/why.md#comparison-chart